src/references.rs

```diff
@@ -85,7 +83,7 @@ fn check_path(
                     )
                 })
                 .collect_vec()
-                .with_context(|| format!("Error while recursing into {}", subpath))?,
+                .with_context(|| format!("Error while recursing into {subpath}"))?,
```

Comment:
personally, I prefer non-inlined. Probably out of inertia/habit more than anything. The only substantive comment I have is that you can't do `{struct.field}`, meaning that sometimes you inline, sometimes you don't, bringing some inconsistencies.

Even so, I see spending too much time on this one way or the other as bike-shedding, so in any case, go ahead, and if it becomes an actual issue, we can revisit.